### DISCUSSION TITLE
Solicitação de email para a Python Cerrado

### DISCUSSION THREAD

**Discussion by ericof** [apyb/comunidade#136] [Geral]
Olá,

Para garantir a transição da organização da Python Cerrado desse ano para os anos seguintes, gostaríamos de pedir a criação de uma conta de e-mail pythoncerrado@python.org.br

Abs,
Érico Andrei

**Comment by cecivieira** [comment]
Seguindo padrão dos eventos regionais: cerrado@python.org.br

cc @naanadr

**Comment by naanadr** [comment]
E-mail foi criado, @ericof quando puder me confirma por favor se vocês conseguiram acessar direitinho, e não esqueçam de definir 2FA se não a conta é suspensa 😆

**Reply by ericof** [reply]
Tudo resolvido!
